lib/dialects/postgres/data-types.js

```diff
@@ -55,10 +65,13 @@ var INTEGER = function() {
   BaseTypes.INTEGER.apply(this, arguments);
 
   // POSTGRES does not support any parameters for integer
-  this._length = null;
-  this.options.length = null;
-  this._unsigned = null;
-  this._zerofill = null;
+  if (this._length || this.options.length || this._unsigned || this._zerofill) {
+    this.warn('You used INTEGER with a parameter. This is not supported by POSTGRES. Simple `INTEGER` will be used instead.');
```

Comment:
I'm unhappy with the wording but nothing better came to my mind. The difference I tried to emphasize is between `BLOB('tiny')` and `INTEGER.UNSIGNED`.